pom.xml

```diff
@@ -3210,6 +3210,7 @@
                             <referenceBranch>${env.JAMES_PARTIAL_BUILD_BASE_BRANCH}</referenceBranch>
                             <buildAll>true</buildAll>
                             <skipTestsForNotImpactedModules>true</skipTestsForNotImpactedModules>
+                            <buildSnapshotDependencies>true</buildSnapshotDependencies>
```

Comment:
Well, because they were not built... Leading the build to fail in some case.

We need to depend for tested artifacts on snapshot we built.

From https://github.com/lesfurets/partial-build-plugin:

```
buildSnapshotDependencies:

Builds all dependencies that are in SNAPSHOT versions. This is necessary if you are using partial build on a reactor where multiple SNAPSHOT projects depend on each other.
```

Given that all of our build consist of snapshot dependencies, turning this on seems a good idea....